bcache/get.go

```diff
@@ -22,11 +22,35 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+
+	"github.com/prometheus/procfs/sysfs"
 )
 
-// ReadStats retrieves bcache runtime statistics for each bcache.
-func ReadStats(sysfs string) ([]*Stats, error) {
-	matches, err := filepath.Glob(path.Join(sysfs, "fs/bcache/*-*"))
+// Bcache represents the pseudo-filesystem proc, which provides an interface to
+// kernel data structures.
+type Bcache struct {
```

Comment:
I'm mobile and will do more review tomorrow, but naming not on these changes: bcache.Bcache and the like are stuttering names which are usually avoided in Go. Same with bcache.NewBcache.

How about calling these types "Handle" and the constructors just "New"?

bcache.Handle, bcache.New
xfs.Handle, xfs.New
Etc.

Will give a closer look later.

Reply:
Yeah, makes sense.  I'm wondering if I should just call them FS and NewFS to match the main procfs and sysfs modules.